examples/open-source-101/test/development.spec.ts

```diff
@@ -0,0 +1,65 @@
+import supertest from 'supertest';
+import { WireMock } from 'wiremock-captain';
+import spotifyGetArtistResponse from './../src/spotify-get-artist-200-resp.json';
+
+const app = supertest('http://localhost:8080');
+
+describe('/artist-popularity', () => {
+    const mockServer = new WireMock('http://localhost:8085');
+
+    afterEach(() => {
+        // clear mocks after each test
+        return mockServer.clearAllExceptDefault();
+    });
+
+    test('should respond back with successful response when spotify API is working', async () => {
+        // register successful Spotify API response
+        await mockServer.register(
+            // incoming request schema
+            {
+                method: 'GET',
+                endpoint: '/v1/artists/0TnOYISbd1XYRBk9myaseg',
+            },
+            // error response schema
+            {
+                status: 200,
+                body: spotifyGetArtistResponse,
+            },
+        );
+
+        const resp = await app.get('/artist-popularity');
+        expect(resp.statusCode).toEqual(200);
+        expect(resp.body).toBeDefined();
+        expect(resp.body.artistName).toBeDefined();
+        expect(resp.body.artistPopularity).toBeDefined();
+        expect(resp.body.artistGenres).toBeDefined();
+        expect(resp.body.artistGenres.length).toBeGreaterThan(0);
+    });
+
+    // why use wiremock-captain
```

Comment:
did you mean to leave this comment here? What I mentioned earlier btw was just making it clear that there are two different use cases that people can incorporate with the mocks: 

1. Mocking a successful happy path that returns expected response 
2. Mocking a failure path that returns 401 

Like maybe emphasize(either when you're presenting or in a comment) that it's simple to change the response and status code and perform different scenarios. Like the 2 cases are basically identical except for a few changes.

Reply:
makes sense. thanks for the feedback!